docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -1093,3 +1093,33 @@ Here's how to migrate:
    },
  },
 ```
+
+## Tooltip
+
+Use the [codemod](https://github.com/mui/material-ui/tree/HEAD/packages/mui-codemod#tooltip-props) below to migrate the code as described in the following sections:
+
+```bash
+npx @mui/codemod@latest deprecations/tooltip-props <path>
+```
+
+### components
+
+The Tooltip's `components` was deprecated in favor of `slots`:
```

Comment:
```suggestion
The Tooltip's `components` prop was deprecated in favor of `slots`:
```

Reply:
That word is missing for other component docs in the file. I will update my changes only